src/internal/BehaviorSubject.ts

```diff
@@ -21,25 +19,22 @@ export class BehaviorSubject<T> extends Subject<T> {
   }
 
   /** @deprecated This is an internal implementation detail, do not use. */
-  _subscribe(subscriber: Subscriber<T>): Subscription {
+  protected _subscribe(subscriber: Subscriber<T>): Subscription {
     const subscription = super._subscribe(subscriber);
-    if (subscription && !(<SubscriptionLike>subscription).closed) {
-      subscriber.next(this._value);
-    }
+    !subscription.closed && subscriber.next(this._value);
```

Comment:
Minifiers usually do this compression for us. I'm not personally a fan of hand writing conditional invocations like this since it makes it non-obvious, to me, when scanning the code. YTMMV

Reply:
I opted to specifically "self minify" here, just to _be sure_ that this would be the minified outcome, rather than best-guessing what the tooling would do.